### PR TITLE
add about content

### PR DESCRIPTION
**Pull request by maelle**
Feedback more than welcome now that I know how to edit content easily :wink:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+# Notes on how to build the website 
+
+## On macOS
+
+```
+npm install
+npm install -g  supervisor
+export JENKINS_URL=http://jenkins.r-hub.io:8080/
+export  LOGDB_URL=http://172.31.30.118:5984/logs
```
Comment (gaborcsardi) [inline README.md]: Can you please change these to 
```
export JENKINS_URL=http://foo.bar
export LOGDB_URL=http://foo.bar
```
and now we also need 
```
MAILGUN_API_KEY=key
```

But I could actually just  make all these optional.... that's what  I'll do, including redis.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+## On Windows
+
+```
+# first install npm
+npm install
+npm install -g  supervisor
+SET JENKINS_URL=http://jenkins.r-hub.io:8080/
+SET LOGDB_URL=http://172.31.30.118:5984/logs
```
Comment (gaborcsardi) [inline README.md]: Same here.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+# In another terminal:
+redis-server
+# In the original terminal:
+supervisor bin/www
+# browse http://localhost:3000/
+
+```
+
+
+# How to create the about
```
Comment (gaborcsardi) [inline README.md]: I don't think we need this is the README. Maybe in the file itself as comments?

File: views/aboutdiv.hjs [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<h1>About the R-hub builder</h1>
+<p>The R-hub builder offers free R CMD Check as a service on different platforms. It is a project by Gábor Csárdi, supported by the R Consortium.</p>
```
Comment (gaborcsardi) [inline views/aboutdiv.hjs]: Please remove my name and  just say that this is an R Consortium project.

File: views/aboutdiv.hjs [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+<h2>Why use the R-hub builder?</h2>
+<p>The builder allows you to check your R package on several platforms, and R versions.</p>
+<p>Moreover, as a side-effect it also allows you to <em>build</em> your R package on several platforms, and R versions.</p>
```
Comment (gaborcsardi) [inline views/aboutdiv.hjs]: We should mention that "buiding" means building binaries, and building binaries for all  (source) dependencies as well. Maybe?

File: views/aboutdiv.hjs [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+<p>You can use the R-hub builder either</p>
+<ul>
+<li>
+<p>via the website (where you are now!), from the <a href="https://builder.r-hub.io/">main page</a> or <a href="https://builder.r-hub.io/advanced">the advanced page</a>.</p>
+</li>
+<li>
+<p>via its API, in particular by using the <a href="https://github.com/r-hub/rhub"><code>rhub</code> package</a>.</p>
+</li>
+</ul>
+<p>You can see a live demo of both the website frontend and of the <code>rhub</code> R package by Gábor Csárdi in <a href="https://www.r-consortium.org/events/2016/10/11/r-hub-public-beta">this video</a>.</p>
```
Comment (gaborcsardi) [inline views/aboutdiv.hjs]: Again, please remove my name. Thanks.

File: views/aboutdiv.hjs [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+<p>Please note that you can only verify your email address from GitHub if the address associated to your GitHub account listed as maintainer address in the DESCRIPTION of the package.</p>
+<h3>Website or package interface to the R-hub builder?</h3>
+<p>Advantages of using the <code>rhub</code> package over the website are that it allows your not leaving R, and that the R package offers shortcut functions such as <code>rhub::check_for_cran</code>, as well as the listing of your recent and current builds, by email address or package.</p>
```
Comment (gaborcsardi) [inline views/aboutdiv.hjs]: `rhub::check_for_cran()`

File: views/aboutdiv.hjs [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+<h2>A bug, question or feature request?</h2>
+<p>Your contributions are welcome. R-hub is developed entirely in the open <a href="https://github.com/r-hub">in its own GitHub organization</a>:</p>
+<ul>
+<li>
+<p>If you can identify to which repo your bug report/feature request/question belongs file an issue there. E.g. the source for this website lives <a href="https://github.com/r-hub/rhub-frontend">here</a>.</p>
```
Comment (gaborcsardi) [inline views/aboutdiv.hjs]: I think we should just forward  everyone  to the `rhub` repo. Simpler

File: views/aboutdiv.hjs [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+</li>
+<li>
+<p>If you can't find to which repo your bug report/feature request/question belongs, no problem, file an issue in <a href="https://github.com/r-hub/rhub">the repo of the R package</a>.</p>
+</li>
+</ul>
+<p>You can also send an email to admin@r-hub.io, and there might soon be a mailing list, stay tuned. You could also ask usage questions over <a href="https://community.rstudio.com/">at RStudio community forum</a>.</p>
```
Comment (gaborcsardi) [inline views/aboutdiv.hjs]: I would remove the  comment about the future mailing list, and also the link  to RStudio community.

**Comment by gaborcsardi** [comment]
I wonder if we should allow writing pages in md.... it is not too hard...

File: README.md (hunk 1 above)
Comment (maelle) [reply README.md]: so should I add "MAILGUN_API_KEY=key"?

File: views/aboutdiv.hjs (hunk 5 above)
Comment (maelle) [reply views/aboutdiv.hjs]: the more details the better!

**Comment by maelle** [comment]
I think I implemented all changes, except the one about the key since I wasn't sure what you meant (whether it'd be optional or not).

Unsurprisingly I like your idea to allow writing pages in md 😁

File: README.md (hunk 1 above)
Comment (gaborcsardi) [reply README.md]: Yes, please. If you rebase you'll see that it is needed.

**Comment by gaborcsardi** [comment]
Thanks! Will see how it looks and then  merge  to master.

**Pull request merged**